tests/rustdoc/inline_cross/impl_trait.rs

```diff
@@ -33,15 +33,7 @@ pub use impl_trait_aux::func4;
 // @!has - '//pre[@class="rust item-decl"]' 'where'
 pub use impl_trait_aux::func5;
 
-// @has impl_trait/fn.async_fn.html
-// @has - '//pre[@class="rust item-decl"]' "pub async fn async_fn()"
-pub use impl_trait_aux::async_fn;
-
 // @has impl_trait/struct.Foo.html
 // @has - '//*[@id="method.method"]//h4[@class="code-header"]' "pub fn method<'a>(_x: impl Clone + Into<Vec<u8, Global>> + 'a)"
 // @!has - '//*[@id="method.method"]//h4[@class="code-header"]' 'where'
 pub use impl_trait_aux::Foo;
-
-// @has impl_trait/struct.Bar.html
```

Comment:
Why removing these checks btw?

Reply:
I've removed them since they've become obsolete with `inline_cross/async-fn.rs` (it also tests async fns). Further, I feel like testing async fns here in `impl_trait.rs` doesn't make sense, it should only test things purely related to impl Trait.
Lastly, the tests didn't catch #115760 since the contains-check `"pub async fn async_fn()"` above failed to notice that `async_fn` was actually "pub async fn async_fn() -> impl Iterator<()>" before this PR.

Reply:
However, I can keep it if you would like.

Reply:
As long as there is an equivalent test, I'm fine with it.